bin/set-macos-keychain.py

```diff
@@ -11,17 +11,15 @@
 # * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
 # * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
 
-__doc__ = """
```

Comment:
Changes in this file are only because it showed flake8 errors when I ran flake8 across the entire project.